src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.model;
+
+import core.basesyntax.ColorSupplier;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double sideA;
+    private final double sideB;
+    private final double height;
+
+    public IsoscelesTrapezoid() {
+        this.sideA = Figure.getRandomNumber();
+        this.sideB = Figure.getRandomNumber();
+        this.height = Figure.getRandomNumber();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-static-methods-in-your-solution)